packages/at_client/lib/src/service/notification_service.dart

```diff
@@ -93,6 +93,7 @@ abstract class NotificationService {
           true, // this was the behaviour before introducing this parameter
       bool checkForFinalDeliveryStatus =
           true, // this was the behaviour before introducing this parameter
+      bool encryptValue = true,
```

Comment:
please add the comment `// this was the behaviour before introducing this parameter`